### PR TITLE
Add modifier

### PR DESCRIPTION
**Pull request by AustinWheel**
### Description

I created a new folder, modifiers, under content/swiftui/concepts. Inside modifiers/, I made a file, modifiers.md, where I wrote the Syntax section as well as an example section.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ - 'Computer Science'
+Tags:
+ - 'SwiftUI'
+ - 'SwiftUI Modifiers'
+CatalogContent:
+ - 'learn-swift'
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [Views](https://www.codecademy.com/resources/docs/swiftui/views).
```
Comment (SSwiniarski) [inline content/swiftui/concepts/modifiers/modifiers.md:15]:
```suggestion
In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        }
+    }
+}
+```
+
+Here the `bold(_:)` `modifier` is being called on the text: `Text("Hello World")`, which, replaces the original "Hello World" text with a new "**Hello World**" text. In this way the `modifier` method `bold(_:)` has been called on a view, and returned a new, altered view.
+
+## Example
+
+The following example displays a text, "Hello Sam", where nameText is the name of a user, and contains a button that increases `codingStreakCount` each time that the Button is pressed.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/modifiers/modifiers.md:37]: I think we'd prefer a simpler example just to demonstrate the use of modifiers in particular. And the introduction to the example should not just explain what the example does, but relate it to the subject of the entry.

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        .padding()
+    }
+}
+```
+
+Each time a user presses the Button, the variable `codingStreakCount` will increase by one.
+
+This code creates this output:
+
+![alt text](https://github.com/AustinWheel/firstRep/blob/main/ExampleCode.png)
```
Comment (SSwiniarski) [inline content/swiftui/concepts/modifiers/modifiers.md:69]: Images should be uploaded to the docs/media folder, and any imbedded images need to link there. You also need to change the alt text of your example image to something descriptive.

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+CatalogContent:
+ - 'learn-swift'
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [Views](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+A `modifier` is a method that when called on a `View`, returns a new, altered `View`, that will replace the original `View` created. Built in `modifiers` such as `bold(_:)`, come built-in with Swift and all `Views` have access to these methods. `bold(_:)` can be used like so:
```
Comment (SSwiniarski) [inline content/swiftui/concepts/modifiers/modifiers.md:19]:
```suggestion
A `modifier` is a method that when called on a `View`, returns a new, altered `View`, that will replace the original `View` created. Built in `modifiers` such as `.bold()`, come built-in with Swift and all `Views` have access to these methods. `.bold()` can be used like so:
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        VStack {
+            Text("Hello World")
+                .bold()
+                //Modifier methods like .bold() can be called underneath a text, button or even a View.
+        }
+    }
+}
+```
+
+Here the `bold(_:)` `modifier` is being called on the text: `Text("Hello World")`, which, replaces the original "Hello World" text with a new "**Hello World**" text. In this way the `modifier` method `bold(_:)` has been called on a view, and returned a new, altered view.
```
Comment (SSwiniarski) [inline content/swiftui/concepts/modifiers/modifiers.md:33]:
```suggestion
Here the `.bold()` `modifier` is being called on the text: `Text("Hello World")`, which, replaces the original "Hello World" text with a new "**Hello World**" text. In this way the `modifier` method `.bold()` has been called on a view, and returned a new, altered view.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@AustinWheel, I did a first review and left a few comments.

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+ - 'Computer Science'
+Tags:
+ - 'SwiftUI'
+ - 'SwiftUI Modifiers'
+CatalogContent:
+ - 'learn-swift'
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:15]:
```suggestion
In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+```psuedo
+struct MyView: View {
+    var body: some View {
+        VStack {
+            Text("Hello World")
+                .bold()
+                //Modifier methods like .bold() can be called underneath a text, button or even a View.
+        }
+    }
+}
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:30]:
```suggestion
struct MyView: View {
  var body: some View {
      VStack {
          Text("Hello World")
              .bold()
              // Modifier methods like .bold() can be called underneath a text, button or even a View.
      }
  }
}
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+            }
+            .padding()
+            .overlay(
+                RoundedRectangle(cornerRadius: 16)
+                    .stroke(.brown, lineWidth: 4))
+        }
+        .foregroundColor(.brown)
+        .padding()
+    }
+}
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:62]:
```suggestion
struct ContentView: View {
  var nameText: String = "Sam"
  @State var codingStreakCount = 0
  var body: some View {
    VStack {
      Text("Hello \(nameText)")
        .font(.title)
        .bold()
        .multilineTextAlignment(.center)
      Button("Coding Streak: \(codingStreakCount)") {
        self.codingStreakCount += 1
      }
      .padding()
      .overlay(
        RoundedRectangle(cornerRadius: 16)
          .stroke(.brown, lineWidth: 4)
      )
    }
    .foregroundColor(.brown)
    .padding()
  }
}
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+Each time a user presses the Button, the variable `codingStreakCount` will increase by one.
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:65]:
```suggestion
Each time a user selects the `Button`, the value of the `codingStreakCount` variable will increase by one.
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+A `modifier` is a method that when called on a `View`, returns a new, altered `View`, that will replace the original `View` created. Built in `modifiers` such as `.bold()`, come built-in with Swift and all `Views` have access to these methods. `.bold()` can be used like so:
+
+```psuedo
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:21]:
```suggestion
```pseudo
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 11]
```diff
@@ -0,0 +1,69 @@
...
+CatalogContent:
+ - 'learn-swift'
+ - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), **Modifiers** are methods that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views).
+
+## Syntax
+
+A `modifier` is a method that when called on a `View`, returns a new, altered `View`, that will replace the original `View` created. Built in `modifiers` such as `.bold()`, come built-in with Swift and all `Views` have access to these methods. `.bold()` can be used like so:
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:19]: The first sentence on this line should be moved up with the text on line # 15. The other sentence should be moved below the \```pseudo code block, with the text on line # 33.

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 12]
```diff
@@ -0,0 +1,69 @@
...
+        VStack {
+            Text("Hello World")
+                .bold()
+                //Modifier methods like .bold() can be called underneath a text, button or even a View.
+        }
+    }
+}
+```
+
+Here the `.bold()` `modifier` is being called on the text: `Text("Hello World")`, which, replaces the original "Hello World" text with a new "**Hello World**" text. In this way the `modifier` method `.bold()` has been called on a view, and returned a new, altered view.
```
Comment (Dusch4593) [inline content/swiftui/concepts/modifiers/modifiers.md:33]:
```suggestion
Here, the `.bold()` modifier is called on the "Hello World" string in the `Text` view, which, replaces the original text with a bolded version of it. This ultimately returns a new, altered `Text` view.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@AustinWheel Finished with my review and left you some comments/suggestions.

@yangc95 @SSwiniarski I wanted to check with you if we should also include information about the `ViewModifier` protocol in this entry? (per [this source](https://developer.apple.com/documentation/swiftui/viewmodifier))

**Comment by SSwiniarski** [comment]
@Dusch4593, that makes sense.

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 13]
```diff
@@ -0,0 +1,69 @@
+---
+Title: 'Modifiers'
```
Comment (yangc95) [inline content/swiftui/concepts/modifiers/modifiers.md:2]:
```suggestion
Title: 'ViewModifier'
```

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+Description: 'Modifiers are methods that can alter views.'
```
Comment (yangc95) [inline content/swiftui/concepts/modifiers/modifiers.md:3]:
```suggestion
Description: 'ViewModifiers are methods that can alter views.'
```

File: content/swiftui/concepts/modifiers/modifiers.md (hunk 6 above)
Comment (yangc95) [reply content/swiftui/concepts/modifiers/modifiers.md:15]: I updated this to display the correct term to use here,

File: content/swiftui/concepts/modifiers/modifiers.md (hunk 11 above)
Comment (yangc95) [inline content/swiftui/concepts/modifiers/modifiers.md:19]:
```suggestion
```

File: content/swiftui/concepts/modifiers/modifiers.md (hunk 8 above)
Comment (yangc95) [reply content/swiftui/concepts/modifiers/modifiers.md:62]: @Dusch4593 does the 2-space indent apply to Swift as well? In Xcode, everything is automatically tabbed so it's better to copy/paste the examples here into their own project just to see.

File: content/swiftui/concepts/modifiers/modifiers.md [hunk 15]
```diff
@@ -0,0 +1,69 @@
...
+        .padding()
+    }
+}
+```
+
+Each time a user presses the Button, the variable `codingStreakCount` will increase by one.
+
+This code creates this output:
+
+![alt text](https://github.com/AustinWheel/firstRep/blob/main/ExampleCode.png)
```
Comment (yangc95) [inline content/swiftui/concepts/modifiers/modifiers.md:69]:
```suggestion
![alt text](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-modifier.png)
```

File: content/swiftui/concepts/viewmodifier/viewmodifier.md [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+            .padding()
+    }
+}
+```
+
+Each time a user clicks the `Button` labeled `"Coding Streak:"`, the value of the `codingStreakCount` variable will increases by one.
+
+This code creates this output:
+
+![SwiftUI Modifier](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-modifier.png)
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/viewmodifier.md:65]:
```suggestion
![SwiftUI Modifier](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-modifier.gif)
```

**Review by yangc95 — APPROVED** [review]
@AustinWheel @Dusch4593 @SSwiniarski I added in a gif for this entry, and re-worked the codeblocks as well as certain wordings. Hope to get this merged soon! TY for everyone's patience 🙂

Edit: I am also going to create a separate issue on adding another section called "Custom ViewModifiers" to this entry once merged, in addition to a number of modifier term issues.

File: content/swiftui/concepts/modifiers/modifiers.md (hunk 2 above)
Comment (yangc95) [reply content/swiftui/concepts/modifiers/modifiers.md]: @SSwiniarski I adjusted the example and image to match

cc: @AustinWheel

File: content/swiftui/concepts/viewmodifier/viewmodifier.md [hunk 17]
```diff
@@ -0,0 +1,51 @@
...
+        // View(s) here
+            // Modifier methods like .bold() can be called underneath a View.
+            // Multiple modifiers can be chained using dot notation.
+    }
+}
+```
+
+## Example
+
+The following example displays some text.
```
Comment (Dusch4593) [inline content/swiftui/concepts/viewmodifier/viewmodifier.md:31]:
```suggestion
The following example displays some text:
```

File: content/swiftui/concepts/viewmodifier/viewmodifier.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
...
+
+    var body: some View {
+        Text("Hello \(name)")
+            .font(.title)
+            .bold()
+    }
+}
+```
+
+In the example above, the `.font()` and `.bold()` modifiers are called on a `Text` `View`. In addition, a built-in text style, `.title` property, is passed into the `.font` modifier.
```
Comment (Dusch4593) [inline content/swiftui/concepts/viewmodifier/viewmodifier.md:47]:
```suggestion
In the example above, the `.font()` and `.bold()` modifiers are called on a `Text` view. In addition, a built-in text style, `.title` property, is passed into the `.font` modifier.
```

File: content/swiftui/concepts/viewmodifier/viewmodifier.md [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), a **`ViewModifier`** is a protocol that can be called on a particular [View](https://www.codecademy.com/resources/docs/swiftui/views). Modifier methods adhere to the protocol and return a new, altered `View`, that will replace the originally created `View`.
```
Comment (Dusch4593) [inline content/swiftui/concepts/viewmodifier/viewmodifier.md:15]:
```suggestion
In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), a **`ViewModifier`** is a protocol that can be called on a particular [view](https://www.codecademy.com/resources/docs/swiftui/views). Modifier methods adhere to the protocol and return a new, altered `View`, that will replace the originally created `View`.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@AustinWheel Just three last comments/suggestions from me.

cc: @yangc95 @SSwiniarski

**Review by yangc95 — COMMENTED** [review]
@Dusch4593 helped push up some of these changes!

cc: @AustinWheel

File: content/swiftui/concepts/viewmodifier/viewmodifier.md [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+In [SwiftUI](https://www.codecademy.com/resources/docs/swiftui), a **`ViewModifier`** is a protocol that can be called on a particular [view](https://www.codecademy.com/resources/docs/swiftui/views). Modifier methods adhere to the protocol and return a new, altered `View`, that will replace the originally created `View`.
+
+## Syntax
+
+```pseudo
+struct MyView: View {
+    var body: some View {
+        // View(s) here
+            // Modifier methods like .bold() can be called underneath a View.
+            // Multiple modifiers can be chained using dot notation.
```
Comment (yangc95) [inline content/swiftui/concepts/viewmodifier/viewmodifier.md:24]:
```suggestion
        View(s) here
            Modifier methods like .bold() can be called underneath a View.
            Multiple modifiers can be chained using dot notation.
```

**Review by Dusch4593 — APPROVED** [review]
@AustinWheel This LGTM! 👍🏻

I made a few small changes in 89f163ce404b679ff4e36e7963cc41a056add32d, including:

- Removed link to SwiftUI topic entry from concept file on line # 15.
  - Usually, concepts and terms don't need a link to their parent entry.
- Removed what appeared to be an extra comma.
- On line # 47, added parentheses to second instance of `.font()`.

cc: @yangc95 @SSwiniarski

**Review by SSwiniarski — APPROVED** [review]
@AustinWheel, @Dusch4593 Looks good to me.

**Comment by yangc95** [comment]
Thanks @AustinWheel for this new entry!
Live link: https://www.codecademy.com/resources/docs/swiftui/viewmodifier
Screen Shot 2022-12-09 at 1 00 59 PM

You're currently anonymous, but if you wish to make yourself public, add your GitHub username to your Codecademy profile and make it visible to everyone. Please let us know if you have any trouble.

cc: @Dusch4593 @SSwiniarski

**Comment by AustinWheel** [comment]
Thank you for all your help @yangc95, @Dusch4593, and @SSwiniarski! I have added my GitHub username on my Codecademy profile but I'm not sure if I've done it properly, https://www.codecademy.com/profiles/AustinWheel this is my profile if you could let me know if it looks right. I really appreciate all the feedback and help you all offered!

**Comment by Dusch4593** [comment]
Hey @AustinWheel! 👋🏻

Did you try logging out Codecademy profile and back in with your GitHub account?

**Comment by AustinWheel** [comment]
@Dusch4593, I did and that seemed to work! Thank you!

**Pull request merged**